tests/test_asgi.py

```diff
@@ -351,6 +351,15 @@ async def test_websocket_receive(send, receive, message_stack):
 
     assert text == msg["text"]
 
+@pytest.mark.asyncio
+async def test_websocket_bytes_receive(send, receive, message_stack):
+    msg = {"bytes": b"hello", "type": "websocket.receive"}
+    message_stack.append(msg)
+
+    ws = WebSocketConnection(send, receive)
+    bytes = await ws.receive()
```

Comment:
I'll let this slip in tests but using bytes as a variable name is asking for trouble because it replaces the built-in name for type `bytes`

Reply:
I agree. I specified the variable names according to the previous test code.

What about the name of this variable? ``raw``